Makefile

```diff
@@ -34,7 +34,7 @@ neard-release:
 	cargo build -p neard --release
 
 
-debug: neard-debug
+debug:
 	cargo build -p neard
```

Comment:
Sorry about that. Better still (so this target follows other targets),
add:

```
neard-debug:
	cargo build -p neard
```

and remove building `neard` from `debug` target.

Reply:
yeah good point, done.